docs/blog/2019-08-13-Localised-strings-from-Kentico-Cloud-GatsbyJS-GraphQL/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Adding and retrieving localised strings from Kentico Cloud to GatsbyJS and GraphQL
+date: 2019-08-13
+author: "Ilesh Mistry"
```

Comment:
Normally it doesn't matter whether or not we use `"`'s in frontmatter, but in this case we match the author with the info in the `author.yaml` file and with quotes (when I run the site locally) it fails to find the correct author for your name:

```suggestion
author: Ilesh Mistry
```